api/src/lib.rs

```diff
@@ -44,6 +44,8 @@ pub mod routes {
     pub const SET_CHANNEL_FEE: &str = "/v1/channel/setChannelFee";
     /// Close an existing channel with a peer.
     pub const CLOSE_CHANNEL: &str = "/v1/channel/closeChannel/:id";
+    /// Fetch aggregate channel local and remote balances.
+    pub const LOCAL_REMOTE_BALANCE: &str = "/v1/channel/localRemoteBal";
```

Comment:
What is the role of abbreviating long words?

`estimateChannelLiquidity` in line 23 did not abbreviate.

Reply:
not my design, it's taken from here https://github.com/Ride-The-Lightning/c-lightning-REST/blob/master/controllers/localRemoteBal.js